### PR TITLE
incorporate feedback from James

### PR DESCRIPTION
**Pull request by patriknw**
Feedback from @jtownley

### CODE REVIEW

File: docs-source/docs/modules/deployment/pages/aws-install.adoc [hunk 1]
```diff
@@ -103,17 +103,12 @@ The Akka Operator requires a https://docs.aws.amazon.com/eks/latest/userguide/ia
 +
 [source,shell script]
 ----
-eksctl utils associate-iam-oidc-provider --cluster eks-akka-demo --approve
+eksctl utils associate-iam-oidc-provider --cluster eks-akka-demo --region eu-central-1 --approve
```
Comment (patriknw) [inline docs-source/docs/modules/deployment/pages/aws-install.adoc:106]: It's not needed if the user did the `aws configure` as we told them, but for those already having aws installed would probably miss that part. Doesn't hurt to be explicit about it.

File: docs-source/docs/modules/deployment/pages/deploy.adoc [hunk 2]
```diff
@@ -0,0 +1,79 @@
+= Deployment workflow
```
Comment (patriknw) [inline docs-source/docs/modules/deployment/pages/deploy.adoc:1]: We want that they follow the tutorial but some people will just read the pages from the left nav bar top to bottom.

File: docs-source/docs/modules/deployment/pages/aws-ecr.adoc [hunk 3]
```diff
@@ -1,4 +1,4 @@
-= Amazon ECR
+= Amazon Docker repository (ECR)
```
Comment (chbatey) [inline docs-source/docs/modules/deployment/pages/aws-ecr.adoc:1]: `Amazon Elastic Container Registry`

**Review by chbatey — APPROVED** [review]

**Review by jtownley — APPROVED** [review]
Thanks

File: docs-source/docs/modules/deployment/pages/aws-ecr.adoc (hunk 3 above)
Comment (patriknw) [reply docs-source/docs/modules/deployment/pages/aws-ecr.adoc:1]: that's the official name so I can change to that, just worried that nobody understands what it is (which Docker is well known)

Comment (patriknw) [reply docs-source/docs/modules/deployment/pages/aws-ecr.adoc]: I have renamed to Amazon Elastic Container Registry

File: docs-source/docs/modules/deployment/pages/aws-install.adoc (hunk 1 above)
Comment (octonato) [reply docs-source/docs/modules/deployment/pages/aws-install.adoc:106]: I think I have reported it before and I did run `aws configure`. Maybe version related?

**Pull request merged**